src/storm/modelchecker/csl/helper/HybridCtmcCslHelper.cpp

```diff
@@ -317,9 +317,10 @@ std::unique_ptr<CheckResult> HybridCtmcCslHelper::computeBoundedUntilProbabiliti
 
 template<storm::dd::DdType DdType, typename ValueType, typename std::enable_if<!storm::NumberTraits<ValueType>::SupportsExponential, int>::type>
 std::unique_ptr<CheckResult> HybridCtmcCslHelper::computeBoundedUntilProbabilities(
-    Environment const& env, storm::models::symbolic::Ctmc<DdType, ValueType> const& model, bool onlyInitialStatesRelevant,
-    storm::dd::Add<DdType, ValueType> const& rateMatrix, storm::dd::Add<DdType, ValueType> const& exitRateVector, storm::dd::Bdd<DdType> const& phiStates,
-    storm::dd::Bdd<DdType> const& psiStates, bool qualitative, double lowerBound, double upperBound) {
+    Environment const& /*env*/, storm::models::symbolic::Ctmc<DdType, ValueType> const& /*model*/, bool /*onlyInitialStatesRelevant*/,
```

Comment:
I see your point in the distinction between both option. Just FYI: in https://github.com/moves-rwth/storm/issues/154#issuecomment-1078698830, @tquatmann preferred the `maybe_unused` option. I think we should just ensure consistency.

Reply:
The [maybe_unused] is helpful if you have lots of compile-setting dependent inputs (which i think we should avoid anyway). I explicitly silenced some of the mathsat warnings because I think that [[maybe_unused]] is a nuclear option there that avoids any (otherwise helpful) warnings.